### ISSUE TITLE
No CSS source files in npm package

### ISSUE THREAD

**Issue by laurentpayot** [beercss/beercss#11]
Hi, I’m evaluating this project that looks quite impressive. Kudos to you! :beers:

> You can get the latest release using NPM. This release contains source files as well as the compiled CSS and JavaScript files.

Contrary to the above quote from the readme file, there is no CSS source file in the npm package directory:
```
.
├── CHANGELOG.md
├── dist
│   └── cdn
│       ├── beer.min.css
│       └── beer.min.js
├── index.js
├── LICENSE
├── package.json
└── README.md
```

I need to access the `src/shared/` directory so I can create my own version of `src/cdn/beer.css` to select what to import (I don’t want to import `urls.css` as I manage the Roboto font myself and use SVG Material icons instead of a font).

Having `src/cdn/shared/` instead of `src/shared/` and modifying `beer.css` imports accordingly could fix it with the following `package.json` modification:
```json
  "files": [
    "src/cdn",
    "dist/cdn",
    "index.js"
  ]
```
I could do a PR for this but maybe you have a better solution… Just let me know :wink:

**Comment by leonardorafael** [comment]
Nice catch!

**Comment by leonardorafael** [comment]
I released a new version with that (1.2.1). Thanks for help. 😀

**Comment by laurentpayot** [comment]
Awesome. Thanks @leonardorafael ! :+1:
